java/code/src/com/suse/utils/CertificateUtils.java

```diff
@@ -90,4 +96,117 @@ public static Optional<Certificate> parse(InputStream inputStream) throws Certif
         CertificateFactory certificateFactory = CertificateFactory.getInstance("X.509");
         return Optional.of(certificateFactory.generateCertificate(inputStream));
     }
+
+    /**
+     * Saves multiple root ca certificates in the local filesystem
+     *
+     * @param filenameToRootCaCertMap maps filename to root ca certificate actual content
+     */
+    public static void saveCertificates(Map<String, String> filenameToRootCaCertMap) {
+        if ((null == filenameToRootCaCertMap) || filenameToRootCaCertMap.isEmpty()) {
+            return; // nothing to do
+        }
+
+        for (Map.Entry<String, String> pair : filenameToRootCaCertMap.entrySet()) {
+            String fileName = pair.getKey();
+            String rootCaCertContent = pair.getValue();
+
+            if (fileName.isEmpty()) {
+                LOG.error("Illegal empty certificate file name");
+                continue;
+            }
+
+            try {
+                if (rootCaCertContent.isEmpty()) {
+                    CertificateUtils.safeDeleteCertificate(fileName);
+                    LOG.warn("CA certificate file: {} successfully removed", fileName);
```

Comment:
This should be an `info`.
```suggestion
                    LOG.info("CA certificate file: {} successfully removed", fileName);
```

Reply:
ok